### PR TITLE
[13.0][ADD]base_field_deprecated

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
This module adds the possibilty to track all the declared deprecated fields in a fast way.

cc @ForgeFlow

### CODE REVIEW

**Comment by JordiMForgeFlow** [comment]
@GuillemCForgeFlow I think you are missing the static folder with the OCA icon

**Comment by MiquelRForgeFlow** [comment]
@JordiMForgeFlow FYI there is a bot that automatically adds it 😉

**Comment by JordiMForgeFlow** [comment]
> @JordiMForgeFlow FYI there is a bot that automatically adds it wink

Oh I did not know that 😅, thank you @MiquelRForgeFlow! :)

File: base_field_deprecated/models/ir_model_fields.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class IrModelFields(models.Model):
+    _inherit = "ir.model.fields"
+
+    deprecated = fields.Boolean(help="Wheter the field is deprecated or not.")
```
Comment (MiquelRForgeFlow) [inline base_field_deprecated/models/ir_model_fields.py]:
```suggestion
    deprecated = fields.Boolean(help="Whether the field is deprecated or not.")
```

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by JordiMForgeFlow — APPROVED** [review]
LGTM 👍🏼

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**